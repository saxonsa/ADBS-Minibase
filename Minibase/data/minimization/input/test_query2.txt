Q(x) :- R(x,y), R(x,2), R(1,2), R(u,3), R(u,v), S(1,3,4)
