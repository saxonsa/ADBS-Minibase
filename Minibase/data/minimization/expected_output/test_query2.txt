Q(x) :- R(x, 2), R(1, 2), R(u, 3), S(1, 3, 4)
